server/pages/index.js

```diff
@@ -120,7 +121,7 @@ function getDefaultContext( request ) {
 		isFluidWidth: !! config.isEnabled( 'fluid-width' ),
 		abTestHelper: !! config.isEnabled( 'dev/test-helper' ),
 		devDocsURL: '/devdocs',
-		catchJsErrors: '/calypso/catch-js-errors-' + 'v2' + '.min.js',
+		catchJsErrors: '/calypso/catch-js-errors.js'
```

Comment:
If so, we can rename file I guess. The bundler was failing miserably, at least locally.
The other solution I have is just to drop this code in `/index.jade`

I'm really dead set on getting this to prod as simple as possible and iterate later.